warehouse/organizations/models.py

```diff
@@ -417,6 +417,9 @@ def active_subscription(self):
         else:
             return None
 
+    def customer_name(self, site_name="PyPI"):
+        (f"{site_name} Organization -" f"self.display_name ({self.name})"),
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**suggestion:** ‏Correct f string representation here
```suggestion
        (f"{site_name} Organization - {self.display_name} ({self.name})"),
```

Curious why this wasn't caught by test - do we not assert on the outcome of the call of this function somewhere?

Reply:
will add an assertion. good catch(es)